src/pages/VenuePage/VenuePage.tsx

```diff
@@ -100,7 +95,7 @@ const VenuePage: React.FC = () => {
 
     updateUserProfile(user.uid, {
       lastSeenIn: updatedLastSeenIn,
-      lastSeenAt: new Date().getTime(),
+      lastSeenAt: getCurrentTimeInUnixEpochSeconds(),
```

Comment:
Do we want this to be in seconds or milliseconds? As I believe `Date().getTime()` is in milliseconds:

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime
  - > The `getTime()` method returns the number of milliseconds* since the Unix Epoch.
  - > * JavaScript uses milliseconds as the unit of measurement, whereas Unix Time is in seconds.